src/lcm/Q_and_F.py

```diff
@@ -28,60 +28,28 @@
         FloatND,
         InternalUserFunction,
         ParamsDict,
-        Period,
         QAndFFunction,
         RegimeName,
     )
 
 
 def get_Q_and_F(
     regime: Regime,
+    active_periods: dict[str, list[int]],
     internal_functions: InternalFunctions,
     next_state_space_infos: dict[str, StateSpaceInfo],
     grids: dict[RegimeName, Any],
```

Comment:
Changed the order; please adjust like this elsewhere, too. (thinking of `period` like a state)